### ISSUE TITLE
Under certain conditions, any changes to a test will not take affect until environment is reset

### ISSUE THREAD

**Issue by alekdavisintel** [AnWeber/vscode-httpyac#306]
I was helping a co-worker yesterday and noticed that while debugging some requests, any changes to the test (like adding or removing assserts) did not get reflected. I did not pay attention but today the very same thing happened to me.

At certain point, once I initially ran a test, the results were always the same for the consequitive runs regarless of the changes I applied. Like if the initial test was successful, and then I changed or added asserts that should've caused errors, they did not. I even changed the endpoint URL (appended random characters at the end) and the test would always show the same (original) result. And if I originally had an error, fixing the assert conditions did not do anything either: it always showed the same error. Once I reset the environment, it all started working, but again only for the first run.

I restarted VSCode, and the problem went away, but it definitely happened on a few occasions, although I cannot say what triggered it. Any ideas what it could be?

**Comment by AnWeber** [comment]
I would need more information for the behavior, I could not understand the behavior and it will be difficult to make progress here

**Comment by alekdavisintel** [comment]
I cannot duplicate it and I have not seen it happened since I submitted this issue, so I will close it. Could be a VSCode issue of some sort (I do not use VSCode much, mostly for httpYac projects and an occasional PowerShell dev work, and I noticed a few times it was doing weird stuff, but maybe it was me doing, not sure). Anyway, thanks for checking, but I think we're good for now. If I notice it again and can duplicate it, I will reopen.
